.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - main
+
+name: docker-build
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Why do we need qemu?

Reply:
Whoops, we don't need it (copy/paste problem)!

But we _do_ need your admin privileges to create the corresponding repo on docker hub and add the `DOCKERHUB_USERNAME`/ `DOCKERHUB_TOKEN` secrets to this repo.

Reply:
Done